src/Exception/BadRequestTokenHttpException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+/**
+ * This exception is thrown when there is an issue with the request token.
+ *
+ * FIXME: obsolete when using symfony security?
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class BadRequestTokenHttpException extends BadRequestHttpException
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(
+        $message = 'Invalid request token. Please <a href="javascript:window.location.href=window.location.href">go back</a> and try again.',
```

Comment:
It is 1:1 copied from https://github.com/contao/core/blob/5d878de67eb218d5eb7ba967dbc669859b8489ec/system/initialize.php#L271
What do you suggest to change it to?